public/app/plugins/datasource/tempo/traceql/autocomplete.ts

```diff
@@ -337,11 +337,15 @@ export class CompletionProvider implements monacoTypes.languages.CompletionItemP
           type: 'OPERATOR',
         }));
       case 'SPANSET_PIPELINE_AFTER_OPERATOR':
-        return CompletionProvider.functions.map((key) => ({
+        const functions = CompletionProvider.functions.map((key) => ({
           ...key,
           insertTextRules: this.monaco?.languages.CompletionItemInsertTextRule?.InsertAsSnippet,
-          type: 'FUNCTION',
+          type: 'FUNCTION' as CompletionType,
```

Comment:
@fabrizio-grafana we can use `as const` here instead of the type assertion, see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions